### PR TITLE
Provide mechanism for accessing generated uarts

### PR DESCRIPTION
**Pull request by jerryz123**

### CODE REVIEW

File: src/main/scala/devices/uart/UARTTx.scala [hunk 1]
```diff
@@ -29,11 +29,12 @@ class UARTTx(c: UARTParams) extends Module {
   io.out := out
 
   val plusarg_tx = PlusArg("uart_tx", 1, "Enable/disable the TX to speed up simulation").orR
+  val plusarg_printf = PlusArg("uart_tx_printf", 0, "Enable/disable the TX").orR
```
Comment (ZenithalHourlyRate) [inline src/main/scala/devices/uart/UARTTx.scala]: Should this default to 1? Otherwise the behavior will change for end users.

Comment (jerryz123) [reply src/main/scala/devices/uart/UARTTx.scala]: Ok

**Comment by jerryz123** [comment]
Please MERGE commit, since main is broken

**Review by abejgonzalez — APPROVED** [review]
LGTM.

**Pull request merged**